java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Split a query to the DB when the parameters count is higher than a certain limit
```

Comment:
This is written from the POV of a developer. What is the user-visible effect of this change?

Reply:
right, probably we don't need a change log here

Reply:
Is it true that we expect more reliability in case certain pages are visited and many systems are registered? Could we use that as an explanation?